latvian/grab-lighter/grab-lighter.md

```diff
@@ -2,4 +2,7 @@ Tu ar trīcošām rokām paķer šķiltavas no zemes.
 
 Tu drudžaini tās mēģini noslēpt savā bikšū kabatā.
 
-Tu izvelc telefonu no otras kabatas un Googlei jautā: ko tagad darīt?
\ No newline at end of file
+Tu izvelc telefonu no otras kabatas un Googlei jautā: ko tagad darīt?
+
+Tacu pirms paradas atbilde, telefons izladejas. Shis fakts tevi
+ashi atgriezh [realitate](es_aizeju.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.